lib/web/apiserver.go

```diff
@@ -1850,6 +1851,15 @@ func (h *Handler) getWebConfig(w http.ResponseWriter, r *http.Request, p httprou
 		}
 	}
 
+	isPolicyRoleVisualizerEnabled := true
+	if roleVisualizerFlag := os.Getenv("TELEPORT_UNSTABLE_ENABLE_ROLE_VISUALIZER"); roleVisualizerFlag != "" {
+		isPolicyRoleVisualizerEnabled, err = strconv.ParseBool(roleVisualizerFlag)
+		if err != nil {
+			h.logger.WarnContext(context.Background(), "unexpected value for TELEPORT_UNSTABLE_ENABLE_ROLE_VISUALIZER environment variable", "error", err)
+			isPolicyRoleVisualizerEnabled = true // default to true
```

Comment:
I decided to "fallback" on the default of true if this env var is set incorrectly. however, since the only reason they would set this var is to be "false", perhaps we log the error that its an invalid value and still set to false? i can go either way

Reply:
Seems fine to me, though if you invert it you don't even need to check the error as `ParseBool` always returns false on error.

```
disableRoleVisualizer, _ = strconv.ParseBool("TELEPORT_UNSTABLE_DISABLE_ROLE_VISUALIZER")
```